specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/definitions.json

```diff
@@ -2334,6 +2352,30 @@
         }
       }
     },
+    "BackendPool": {
+      "type": "object",
+      "description": "Backend pool information",
+      "properties": {
+        "services": {
+          "type": "array",
+          "description": "The list of backend entities belonging to a pool.",
+          "items": {
+            "$ref": "#/definitions/BackendPoolItem"
+          },
+          "minItems": 2
+        }
+      }
+    },
+    "BackendPoolItem": {
+      "type": "object",
+      "description": "Pool service information",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The unique id of the backend entity. The ID should refer to an already existing backend entity."
```

Comment:
@m-h-s is it the ARM Id of Entity name.

Can you add an example so it is clear

Reply:
Can you please add examples as well so that it is clear for review the change